INSTALL.md

```diff
@@ -59,13 +59,12 @@ but for most cases this is not recommended.
 Then run:
 
     $ docker-compose pull
-    $ docker-compose up
+    $ docker-compose up --detach
```

Comment:
```suggestion
    $ docker-compose up -d
```
I know you like the verbose versions of commands more, but for the install documentation I would suggest the shorthand for an easier usage.